composer.json

```diff
@@ -18,5 +19,10 @@
         "phar": {
             "bundler": "composer"
         }
+    },
+    "config": {
+        "platform": {
+            "php": "5.3.6"
```

Comment:
and this one is used for lock file (so you won't lock to version of dependency that works on 5.6 but not 5.4)